test/functional/http/get/transactions.js

```diff
@@ -32,8 +32,8 @@ describe('GET /api/transactions', function () {
 	before(function () {
 
 		var promises = [];
-		promises.push(creditAccountPromise(account.address, 100*100000000));
-		promises.push(creditAccountPromise(account2.address, 20*100000000));
+		promises.push(creditAccountPromise(account.address, 100 * 100000000));
```

Comment:
Use `node.normalizer` (name is not the best one) or maybe something from `constants` for calculations